### DISCUSSION TITLE
More companies with remote jobs

### DISCUSSION THREAD

**Discussion by douglasdcm** [remoteintech/remote-jobs#1631] [General]
Hi, I hope you are all good. Fist of all, congratulations for creating this amazing repository. It is very helpful. To contribute, I made a list of new companies that have remove jobs. I checked and none of them are in README.md. I think they could be added to this repository. I added one today, but it take some time, so if someone wants to help, it is welcome :)

To check the companies jobs, go to [source](https://vagaspramim.onrender.com) and search for "remote"

- https://3m.recsolu.com
- https://atento.gupy.io
- https://careers.accor.com
- https://careers.mars.com
- https://carreiras.dell.com
- https://cesar.breezy.hr
- https://cielo.gupy.io
- https://embraer.gupy.io
- https://eurofarma.gupy.io
- https://gruposabin.gupy.io
- https://jobs.quickin.io
- https://jobs.sap.com
- https://oi.gupy.io/job
- https://petlove.gupy.io
- https://pmweb.gupy.io
- https://takeblip.gupy.io
- https://tech-career.gupy.io
- https://unidas.gupy.io
- https://vempra.gupy.io
- https://vivo.gupy.io
- https://wiz.gupy.io
- https://www.avenuecode.com.br
- https://www.dqrtech.com.br
- https://www.messagebird.com
- https://www.nerdwallet.com

**Comment by Ki-er** [comment]
I have created an issue for this.
